tg_react/templates/emails/base.txt

```diff
@@ -0,0 +1,5 @@
+Hi there,\n
```

Comment:
Can you prefix the templates folder with `tg_react` so we don't collide with other packages.